src/main/kotlin/domain/Direction.kt

```diff
@@ -0,0 +1,18 @@
+package domain
+
+enum class Direction(val rowDirection: Int, val columnDirection: Int) {
+    TOP(-1, 0),
+    TOP_RIGHT(-1, 1),
+    RIGHT(0, 1),
+    BOTTOM_RIGHT(1, 1),
+    BOTTOM(1, 0),
+    LEFT(0, -1),
+    TOP_LEFT(-1, -1);
+
+    fun getSurroundLocation(location: Location): Location? {
```

Comment:
```suggestion
    fun getSurroundLocation(location: Location): Location {
```

null 이 될 수 있음을 제거해볼까요?